app/routes.py

```diff
@@ -0,0 +1,334 @@
+from app import db
+from app.models.customer import Customer
+from flask import Blueprint, jsonify, make_response, request
+from datetime import datetime, date, timedelta
+from app.models.video import Video
+import requests, json
+from dotenv import load_dotenv
+import os
+from app.models.utility_func import *
+from app.models.rental import Rental
+
+
+customer_bp = Blueprint("customer", __name__, url_prefix="/customers")
+video_bp = Blueprint("video", __name__, url_prefix="/videos")
+rental_bp = Blueprint("rental", __name__, url_prefix="/rentals")
+
+@customer_bp.route("", methods=["GET"])
+def get_all_customers():
+    all_customers = Customer.query.all()
+
+    if not all_customers:
+        return jsonify([]), 200
+    
+    customers_list = [] 
+    for customer in all_customers:
+        customer_dict = {
+            "id" : customer.id, 
+            "name" : customer.name, 
+            "postal_code" : customer.postal_code,
+            "phone": customer.phone,
+            "register_at" : customer.register_at
+        }
```

Comment:
you created an instance method in your Customer model to handle this. I would use it here.